mpas_analysis/polar_regions.cfg

```diff
@@ -425,6 +460,55 @@ makeTables = True
 iceShelvesInTable = ['all']
 
 
+[timeSeriesOceanRegions]
+## options related to plotting time series of groups of ocean regions
+
+# the names of region groups to plot, each with its own section below
+regionGroups = ['Arctic Ocean Regions']
```

Comment:
Instead of this section here, I think we just want to add "Arctic Ocean Regions" to `regionGroups` in `default.cfg`.

Reply:
So does this mean that we want Arctic Ocean regional time series on by default and the rest on only with the `polar_regions` flag?

Reply:
I see.  I missed that nuance.  The problem with what is here now is that it seems like it switches from Antarctic regions to Arctic Ocean regions when `--polar_regions` is specified.  I think we want the behavior to be that neither is used by default and both are used with `--polar_regions`.  So maybe we want `regionGroups = []` in `default.cfg` and `regionGroups = ['Antarctic Regions', 'Arctic Ocean Regions']` here.

However, it is still the case that the `[timeSeriesArcticOceanRegions]` section should be defined in `default.cfg` instead of here so that someone could add:
```
[timeSeriesOceanRegions]
regionGroups = ['Arctic Ocean Regions']
```
to their own config file without having to specify `--polar_regions` and would get the Arctic Ocean Regions.  Does that make sense?  If not, let's chat about it tomorrow.

Reply:
I think that makes sense, but then I would question whether we should do the same thing for `TSDiagramsForArcticOceanRegions` analogously to `TSDiagramsForAntarcticRegions`.

Reply:
Yes, I think so. For as much as possible, the config options in the default file should make sense and be as comprehensive as possible even if the analysis is off or doesn't include that region by default. The polar regions config file should then turn things on and maybe tweak them slightly.

Feel free to clean things up as you see fit. I'll try to make sure the results in zppy with the WC configuration is roughly unchanged.